### PR TITLE
feat(radii): extend radii theme object with common values

### PR DESCRIPTION
**Pull request by dclerici-ssc**
- border radious new values are extracted
- usages of 'getBorderRadius' are replaced with 'getRadii' method
- custom border-radius values are replaced with helper method if applicable

Also, it fixes a border-radius bug on the segmentedToggleItem we had. 

[Closes ](https://zitenote.atlassian.net/browse/UXD-391)

### CODE REVIEW

File: src/components/IconButton/IconButton.tsx [hunk 1]
```diff
@@ -4,15 +4,15 @@ import styled from 'styled-components';
...
+import { getFontSize, getRadii, pxToRem } from '../../utils';
 import { Button } from '../Button';
 import { ButtonVariants } from '../Button/Button.enums';
 import { Icon } from '../Icon';
 import { SSCIcons, Types } from '../Icon/Icon.types';
 import { IconButtonProps } from './IconButton.types';
 
 const StyledIconButton = styled(Button)`
-  border-radius: 100%;
+  border-radius: ${getRadii('circle')};
```
Comment (julesbarr) [inline src/components/IconButton/IconButton.tsx:15]: should we create a enum that holds radii values ?

Comment (ajkl2533) [reply src/components/IconButton/IconButton.tsx:15]: That's good idea, something similar to what we have for colors

Comment (dclerici-ssc) [reply src/components/IconButton/IconButton.tsx:15]: mmm yep.. I was thinking about it when I was doing the replacements. Let's create constants instead

**Review by ajkl2533 — APPROVED** [review]
LGTM

**Pull request merged**